src/controllers/v1.controller.ts

```diff
@@ -185,12 +185,47 @@ export class V1Controller {
     }
 
     try {
-      const loadBalancer = await this.fetchLoadBalancer(id, filter)
+      let loadBalancer = await this.fetchLoadBalancer(id, filter)
 
       if (!loadBalancer?.id) {
         throw new ErrorObject(reqRPCID, new jsonrpc.JsonRpcError('Load balancer not found', -32054))
       }
 
+      const gigastakeOptions: {
+        gigastaked: boolean
+        lbApplication: Applications | undefined
+      } = {
+        gigastaked: loadBalancer.gigastakeRedirect || false,
+        lbApplication: undefined,
+      }
+
+      // Is this LB marked for gigastakeRedirect?
+      // Temporary: will be removed when live
+      if (gigastakeOptions.gigastaked) {
+        for (const redirect of JSON.parse(this.redirects)) {
+          if (this.host.toLowerCase().includes(redirect.blockchain)) {
```

Comment:
If I understand correctly, this is trying to find a `redirect` that has a certain `blockchain` property? Because that `break` on line 224 is a tad smelly haha. I think we can just use `find` on the redirects array.